crates/harness/src/monolith.rs

```diff
@@ -460,4 +471,10 @@ impl MonolithBuilder {
         self.behavior = Some(Box::new(behavior));
         self
     }
+
+    pub fn region(mut self, region: &str) -> Self {
```

Comment:
Would prefer taking in `impl AsRef<str>` instead.
```suggestion
    pub fn region(mut self, region: impl AsRef<str>) -> Self {
```